package io.swagger.exceptions;

public class UserNotFoundException extends RuntimeException{
}
